config/environments/development.rb

```diff
@@ -18,10 +18,13 @@
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
-    config.cache_store = :memory_store
+
     config.public_file_server.headers = {
       'Cache-Control' => "public, max-age=#{2.days.to_i}"
     }
+
+    config.cache_store = :file_store, "#{Rails.root}/tmp/rails-cache"
```

Comment:
Is this going to cache successfully on Heroku? New dynos can spin up without any cache files. The database might be a better place.

Reply:
I've used the db cache quite successfully on https://www.growstuff.org

Reply:
This change is for local development only. Heroku envs will use memcached to store cache